scripts/download-prebuilt.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -e
+
+SETUP_SRC=$(realpath ${BASH_SOURCE[@]})
+SETUP_DIR=$(dirname $SETUP_SRC)
+TOP_DIR=$(realpath $SETUP_DIR/..)
+BUILD_DIR=$TOP_DIR/build
+PREBUILT_DIR=$BUILD_DIR/prebuilt
+
+if [ ! -e $BUILD_DIR ]; then
+	echo "$BUILD_DIR does not exist so it looks like the prerequisit tools may not be available"
+	echo "They can be installed by running scripts/get-env-root.sh and scripts/get-env.sh"
+	exit 1
+fi
+
+: ${BOARD:="atlys"}
+: ${TARGET:="hdmi2usb"}
+: ${PREBUILT_RELEASE:="unstable"}
```

Comment:
I think you should probably be using the version this repository is checked out too rather then a floating version right?